### PR TITLE
what I have so far

### PR DESCRIPTION
**Pull request by tashsmit**
Hey guys, this is what I have so far. The vanilla meme activity works, it writes the text on the top of the image. However I am still trying to figure out how to get the image from main activity to this activity. The code I already have runs but it doesn't work on my phone.

Let me know if you have any questions.

### CODE REVIEW

File: app/src/main/AndroidManifest.xml [hunk 1]
```diff
@@ -14,12 +14,18 @@
         </activity>
         <activity
             android:name=".ChooseMemeStyle"
-            android:label="@string/title_activity_choose_meme_style">
-        <intent-filter>
-            <action android:name="android.intent.action.MAIN" />
+            android:label="@string/title_activity_choose_meme_style" >
```
Comment (poojawins) [inline app/src/main/AndroidManifest.xml:27]: before this is merged, remember to change it back so that MainActivity is the launcher activity.

**Comment by poojawins** [comment]
looks good so far

**Comment by AnthonyFermin** [comment]
Looks good to merge :+1:

**Comment by poojawins** [comment]
landscape for VanillaMeme is breaking but otherwise it works :+1:

**Comment by poojawins** [comment]
I found the error, will commit it now (though the landscape layout is off, but at least it's no longer crashing)

**Pull request merged**